lib/hz2600/Kazoo/Api/Entity/Device.php

```diff
@@ -4,9 +4,10 @@
 
 class Device extends AbstractEntity
 {
-    public function quickcall($number) {
+    public function quickcall($number, $auto_answer = true) {
```

Comment:
Good suggestions. I agree, that's a better way. I've made changes with one modification--- I've set parameters by default to array() to make it optional and not be a breaking change.